src/main/scala/sbtorgpolicies/settings/bash.scala

```diff
@@ -56,15 +57,18 @@ trait bash {
         }
       }.value,
       orgAfterCISuccess := Def.taskDyn {
-        if (getEnvVarOrElse("TRAVIS_BRANCH") == "master" &&
+        val scalaV       = scalaVersion.value
+        val crossV       = crossScalaVersions.value
+        val isLastScalaV = crossV.lastOption.exists(_ == scalaV)
+        if (isLastScalaV &&
+          getEnvVarOrElse("TRAVIS_BRANCH") == "master" &&
```

Comment:
Let's use `orgCommitBranchSetting.value` setting then, instead of hardcoded "master" string